packages/world/ts/node/findSolidityFiles.ts

```diff
@@ -0,0 +1,14 @@
+import path from "node:path";
+import { glob } from "glob";
+import { World } from "../config/v2/output";
+
+export async function findSolidityFiles({ rootDir, config }: { rootDir: string; config: World }) {
```

Comment:
may need to exclude `.t.sol` and `.s.sol`?

Reply:
ah likely because we're all good about putting source in `src` but technically possible to have test files, etc. live in this dir as well (and I personally like colocating tests with the files they're testing)